semgrep-core/src/parsing/pfff/js_to_generic.ml

```diff
@@ -218,9 +218,11 @@ and literal x =
   | String v1 ->
       let v1 = wrap string v1 in
       G.String v1
-  | Regexp v1 ->
+  | Regexp (v1, v2) ->
       let v1 = wrap string v1 in
-      G.Regexp v1
+      (* TODO: *)
```

Comment:
What is left to do exactly?

Reply:
What is left is to change Ast_js.Regexp type to also include position information for the leading / and closing /
and the modifier, so we avoid those fake_bracket and fake_info below.
Otherwise we can match regexps, but the matched position might be wrong if for example
we do /$XXX/g as a pattern, right now $XXX will be bounded to a Parse_info.t that include the whole
range of /abc/g, not just abc.